ansible/roles/machine/setup/tasks/install_packages.yml

```diff
@@ -36,13 +36,19 @@
     - PyYAML  # ipa-vagrant-ci dependency
     - vim
     - NetworkManager
-    - python3-paramiko
```

Comment:
Is the package in fedora dependencies broken? Why is it installed with pip instead?